pkg/timer/tablestore/store.go

```diff
@@ -148,12 +148,27 @@ func (s *tableTimerStoreCore) List(ctx context.Context, cond api.Cond) ([]*api.T
 			timerData = row.GetBytes(3)
 		}
 
+		tz := row.GetString(4)
```

Comment:
Why use a separated `tzParse`, rather than use `tz` directly here?

Reply:
because `tz` will be assigned to `TimerSpec.TimeZone` but `tzParse` is used to parse the real location.

For example, if user created a timer with time zone "" (an empty string, means to use database timezone). The returned `TimerSpec.TimeZone` should also be an empty, but we should use a valid time zone string to parse it to `time.Location`